engine/gamesys/src/gamesys/scripts/script_resource.cpp

```diff
@@ -3102,218 +3102,225 @@ static int GetTextMetrics(lua_State* L)
     return 1;
 }
 
-static const luaL_reg Module_methods[] =
-{
-    {"set", Set},
-    {"load", Load},
-    {"create_atlas", CreateAtlas},
-    {"create_buffer", CreateBuffer},
-    {"create_texture", CreateTexture},
-    {"create_texture_async", CreateTextureAsync},
-    {"release", ReleaseResource},
-    {"set_atlas", SetAtlas},
-    {"get_atlas", GetAtlas},
-    {"set_texture", SetTexture},
-    {"get_texture_info", GetTextureInfo},
-    {"get_render_target_info", GetRenderTargetInfo},
-    {"set_sound", SetSound},
-    {"get_buffer", GetBuffer},
-    {"set_buffer", SetBuffer},
-    {"get_text_metrics", GetTextMetrics},
-    {0, 0}
-};
-
 /*# 2D texture type
```

Comment:
Do we want to keep it in docs? usually we remove deprecated functionality (removing # symbol)

Reply:
Hm, maybe we should just remove the docstrings, we shouldn't need them anymore at all

Reply:
maybe. yes. but we will have to update all the manuals after release of this version